evap/staff/views.py

```diff
@@ -1386,22 +1388,67 @@ def evaluation_email(request, evaluation_id):
     )
 
 
-def helper_delete_users_from_evaluation(evaluation, operation):
-    if "participants" in operation:
-        deleted_person_count = evaluation.participants.count()
-        deletion_message = _("{} participants were deleted from evaluation {}")
-        evaluation.participants.clear()
-    elif "contributors" in operation:
-        deleted_person_count = evaluation.contributions.exclude(contributor=None).count()
-        deletion_message = _("{} contributors were deleted from evaluation {}")
-        evaluation.contributions.exclude(contributor=None).delete()
+class ImportAction(Enum):
+    COPY = "copy"
+    IMPORT = "import"
+    TEST = "test"
+
+    @classmethod
+    def from_operation(cls, operation: str) -> "ImportAction":
+        if "copy" in operation:
+            return cls.COPY
+        if "import" in operation:
+            return cls.IMPORT
+        if "test" in operation:
+            return cls.TEST
+        raise ValueError(f"Unknown operation: {operation}")
```

Comment:
`operation.startswith("copy-")` and similar would allow me not to worry about weird unintended substring matches